activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -156,7 +156,7 @@ def write_entry(key, entry, options)
           expires_in = options[:expires_in].to_i
           if expires_in > 0 && !options[:raw]
             # Set the memcache expire a few minutes in the future to support race condition ttls on read
-            expires_in += 5.minutes
+            expires_in += 300
```

Comment:
This change stems from the `to_i` - the memcache client expects an integer for `expires_in`.  It could be rewritten to use `5.minutes` as the receiver but I thought creating an object was wasteful this deep inside the caching code.